pkg/beholder/message.go

```diff
@@ -211,13 +216,15 @@ func (m *Metadata) FromAttributes(attrs Attributes) *Metadata {
 			m.CapabilityName = v.(string)
 		case "network_chain_id":
 			m.NetworkChainID = v.(string)
+		case "timestamp":
+			m.Timestamp = v.(time.Time)
 		}
 	}
 	return m
 }
 
 func NewMetadata(attrs Attributes) *Metadata {
-	m := &Metadata{}
+	m := &Metadata{Timestamp: time.Now().UTC()}
```

Comment:
@nanchano, We should try not to use attributes with unbounded cardinality. 
It can cause various problems see this thread for details 
https://chainlink-core.slack.com/archives/C06TEMFRPQD/p1729082258236499

Reply:
I could be wrong, but the unbounded cardinality issue is a problem for metrics, not for custom events, right?

Reply:
How would we wanna represent a timestamp then? As a string and let clients parse it?

Reply:
It probably should be inside the custom message payload. 
FYI there is `Timestamp` field exist in otel log record  
https://opentelemetry.io/docs/specs/otel/logs/data-model/#log-and-event-record-definition

> I could be wrong, but the unbounded cardinality issue is a problem for metrics, not for custom events, right?
yes, but attributes can be reused between different telemetry types e.g span attributes are copied to metric attributes.

> not for custom events, right?
I depends how we are going to store it. 

We can put it in the attribute and get back to it later. 
Whats the use case for having it in attributes?

Reply:
Loki labels suffer from hight cardinality as well:
See also https://utcc.utoronto.ca/~cks/space/blog/sysadmin/GrafanaLokiCardinalityProblem
https://grafana.com/docs/loki/latest/get-started/labels/#cardinality

```
This is high cardinality, and it can lead to significant performance degredation.

When we talk about cardinality we are referring to the combination of labels and values and the number of streams they create. High cardinality is using labels with a large range of possible values, such as ip, or combining many labels, even if they have a small and finite set of values, such as using status_code and action.

High cardinality causes Loki to build a huge index and to flush thousands of tiny chunks to the object store. Loki currently performs very poorly in this configuration. If not accounted for, high cardinality will significantly reduce the operability and cost-effectiveness of Loki.
```

Reply:
Right, but custom events are not sent to Loki, so the cardinality issue is at play here, I think